blog/data/authors/andrzej.pawlowicz.json

```diff
@@ -1,5 +1,5 @@
 {
     "name": "Andrzej Pawłowicz",
-    "bio": "",
+    "bio": "Application Architect with over 11 years of professional experience. API designer, C++ advocate experienced in Qt. Former Intel BIOS Software Engineer.",

```

Comment:
@artur-rs  now it is 13, I wonder if it make sense to write that. Maybe `over 10 years` ?

Reply:
```suggestion
    "bio": "Application Architect with over 13 years of professional experience. Application and middleware architect, API designer, modern C++ advocate experienced in Qt. Former Intel BIOS Software Engineer.",
```